zoo-css-challenge/style.css

```diff
@@ -191,5 +208,19 @@ footer {
 /*Programs*/
 
 .programs .card {
-  display: block;
+  .program {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+.card {
+  width: 270px;
+  height: 100px;
+}
```

Comment:
```suggestion
```
None of the tasks were to change the sizes of the `.card` elements, so we can delete this